UnitTestFrameworkPkg/Test/GoogleTest/Sample/SampleGoogleTest/SampleGoogleTest.cpp

```diff
@@ -51,37 +51,46 @@ TEST_F(GlobalBooleanVarTests, GlobalBooleanShouldBeChangeable) {
   pointer is updatable.
 **/
 class GlobalVarTests : public ::testing::Test {
-  public:
-    VOID  *SampleGlobalTestPointer = NULL;
-
-  protected:
-  void SetUp() override {
+public:
+  VOID *SampleGlobalTestPointer = NULL;
+
+protected:
+  void
+  SetUp (

```

Comment:
I'm not a fan of this, personally, not sure why it does this when there are no parameters.

Reply:
It's specialized for edk2 style parameter formatting. Generally, we'd expect `VOID` to be here and it wouldn't be empty.